sudoku/styles.css

```diff
@@ -0,0 +1,107 @@
+.hidden {
+  display: none;
+}
+body.dark {
+  background-color: rgb(45, 100, 117);
+  color: white;
+}
+header {
+  text-align: center;
+  padding-bottom: 20px;
+  padding-top: 10px;
+  border-bottom: 1px solid green;
+}

```

Comment:
Dont target the element directly target it by class or id

Reply:
Target all the element by Id or Class don't target it directly

Reply:
also format the files properly